daft/expressions/expressions.py

```diff
@@ -2445,6 +2445,8 @@ def truncate(self, interval: str, relative_to: Expression | None = None) -> Expr
     def to_unix_epoch(self, time_unit: str | TimeUnit | None = None) -> Expression:
         """Converts a datetime column to a Unix timestamp. with the specified time unit. (default: seconds).
 
+        See :meth:`daft.datatype.TimeUnit` for more information on time units and valid values.
```

Comment:
Why `:meth:` here?

Reply:
oh good call, it should be `:class:` instead.